src/frontend/static/styles/styles.css

```diff
@@ -204,11 +204,14 @@ header .navbar.sub-navbar > .container {
   padding-right: 26px;
 }
 
-header .navbar.sub-navbar .logo {
-  width: 209px;
+header .top-left-logo {
   height: 40px;
```

Comment:
Specifying the `height` also affects the `width` so we don't need to specify the `width` (i.e., the ratio of the image is preserved).

Reply:
Also, I added a CSS class `.top-left-logo` because it's more self-describing than `.navbar.sub-navbar .logo`.